pkg/domain/infra/abi/play_test.go

```diff
@@ -6,34 +6,9 @@ import (
 
 	"github.com/stretchr/testify/assert"
 	v1 "k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
I'm not sure I love this change, why did you do it?

Reply:
whats the concern? the new alias?

Reply:
yeah! v12 doesn't mean very much to me, whereas metav1 (despite not having a ton of meaning) is a little more clear